docs/maui/converters/state-to-bool-converter.md

```diff
@@ -0,0 +1,145 @@
+---
+title: StateToBoolConverter - .NET MAUI Community Toolkit
+author: bijington
+description: "The StateToBoolConverter is a one way converter that returns a boolean result based on whether the supplied value is of a specific LayoutState."
+ms.date: 05/17/2022
+---
+
+# StateToBoolConverter
+
+[!INCLUDE [docs under construction](../includes/preview-note.md)]
+
+The `StateToBoolConverter` is a one way converter that returns a `boolean` result based on whether the supplied value is of a specific `LayoutState`.
+
+The `Convert` method returns a `boolean` result based on whether the supplied value is of a specific `LayoutState`. The `LayoutState` enum is provided by the toolkit and offers the possible values:
+
+- `None`
+- `Loading`
+- `Saving`
+- `Success`
+- `Error`
+- `Empty`
+- `Custom`
+
+> [!NOTE]
+> Note that the expected `LayoutState` can be supplied in the following order of precedence:
+> 1. as the `ConverterParameter` in the converter binding; this supersedes the `StateToCompare` property
+> 2. as the `StateToCompare` property on the converter
+
+The `ConvertBack` method is not supported.
+
+## Syntax
+
+The following example shows how to use the converter to change the visibility of a `Label` control based on the `LayoutState` property which is modified on a `Button` `Command`.ß
```

Comment:
```suggestion
The following example shows how to use the converter to change the visibility of a `Label` control based on the `LayoutState` property which is modified on a `Button` `Command`.
```

Reply:
damn that was my attempt at pressing CMD + S to save the doc but hitting ALT instead. Good catch!